util/cori-knl_setup.sh

```diff
@@ -1 +1,2 @@
-module load python/2.7-anaconda-5.2 git
+module load git
+source /global/project/projectdirs/acme/software/anaconda_envs/load_latest_cime_env.sh
```

Comment:
This broke MPAS builds: https://my.cdash.org/viewTest.php?buildid=1725496. Does that tag need to be updated or a new one made? https://github.com/CVMix/CVMix-src/blob/v0.84-beta/src/shared/makedep.py#L56

Reply:
That MPAS script is python 2 only and the default `cime_env` is python 3. 

IMO the MPAS script should be updated, but a py2 version could be loaded (I think; is there a py2 deployment @xylar?)

Reply:
@amametjanov, what branch is this test running from?  We fixed this in May in MPAS but, to my surprise, it wasn't merged into E3SM until 29 days ago (!!!) but it should be on master now:
https://github.com/MPAS-Dev/MPAS-Model/pull/251
https://github.com/E3SM-Project/E3SM/pull/2952

Reply:
I see that it's using maint-1.0.  I think we may need to patch that branch with a couple of python 3 fixes.